src/braket/ahs/atom_arrangement.py

```diff
@@ -63,56 +69,153 @@ def __init__(self):
 
     def add(
         self,
-        coordinate: Union[tuple[Number, Number], np.ndarray],
+        coordinate: Union[Tuple[Number, Number], np.ndarray],
         site_type: SiteType = SiteType.FILLED,
-    ) -> AtomArrangement:
+    ) -> "AtomArrangement":
```

Comment:
The original hint referred to the `AtomArrangement` class as a return type. What does turning it into a string literal achieve here?